src/open-uri.c

```diff
@@ -689,29 +687,43 @@ handle_open_in_thread_func (GTask *task,
         {
           g_autofree char *real_path = get_real_path_for_doc_path (path, app_id);
 
-          if (open_uri->file_manager != NULL)
```

Comment:
I think you want to remove the immediately following brace pair and dedent all the code within?

Reply:
I've added a second commit which does that.